### PR TITLE
Added Read-Write-Many docs

### PR DESCRIPTION
**Pull request by Pure-AdamuKaapan**
Adds a documentation page for RWX volumes. Also starts the convention of having an examples directory in the docs page, for easier usage of demo files.

### CODE REVIEW

File: docs/csi-read-write-many.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+# Using Read-Write-Many (RWX) volumes with Kubernetes
+
+## Introduction
+
+The Pure Service Orchestrator Kubernetes CSI driver includes support for Read-Write-Many (RWX) volumes starting with version 5.1.0.
```
Comment (sdodsley) [inline docs/csi-read-write-many.md:6]: Makes it sound like we haven't before, even though we have with FlashBlade so maybe say `for FlashArray` somewhere

File: docs/csi-read-write-many.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+|--------------|------------|-------------|--------|
+| Block        | Block      | RWO         | Yes    |
+| Block        | Block      | RWX         | Yes    |
+| Block        | File       | RWO         | Yes    |
+| Block        | File       | RWX         | **No** |
+| File         | File       | RWO         | Yes    |
+| File         | File       | RWX         | Yes    |
+
+## Examples
+To use these examples, install the Pure CSI plugin (either through the operator or helm).
```
Comment (sdodsley) [inline docs/csi-read-write-many.md:25]: let's leave operators out of this... We are probably going to get rid of that deployment method now OpenShift are allowing Helm again

File: docs/csi-read-write-many.md (hunk 1 above)
Comment (Pure-AdamuKaapan) [reply docs/csi-read-write-many.md:6]: Oh thanks, I forgot that we supported it on FB before, I'll fix that.

File: docs/csi-read-write-many.md (hunk 2 above)
Comment (Pure-AdamuKaapan) [reply docs/csi-read-write-many.md:25]: Perfect, that works for me.

**Pull request merged**